package.json

```diff
@@ -17,20 +20,28 @@
   },
   "homepage": "https://github.com/dojo/web-editor#readme",
   "devDependencies": {
+    "@dojo/cli-build-webpack": "^0.2.2",
```

Comment:
We should always semver on `~0.#.0` for the Dojo 2 packages as minor releases introduce breaking changes.